modules/extension/xsd/xsd-gml2/src/test/java/org/geotools/gml2/bindings/GMLLinearRingTypeBindingTest.java

```diff
@@ -212,7 +208,7 @@ public void testCoordinatesLessThanFour() throws Exception {
 
         try {
             s.parse(ring, node, null);
-            fail("Should have thrown an exception with less then 4 points");
```

Comment:
Ah... it has become _more_ tolerant?

Reply:
yep; 
see https://github.com/locationtech/jts/pull/682 ans more specifically https://github.com/locationtech/jts/issues/558#issuecomment-774368579

Reply:
Ah ha... well, that does not seem like something which would cause significant backwards incompatibiliites (besides the oddball test, like this one)